multi/imxrt-multi/spi.c

```diff
@@ -470,116 +466,140 @@ static void spi_initPins(void)
 		int muxes[7];
 	} info[] = {
 #if SPI1
-		{ 0,
-		{ PIN2MUX(SPI1_SCK), PIN2MUX(SPI1_SD0), PIN2MUX(SPI1_SDI),
-#ifdef SPI1_PCS0
-		PIN2MUX(SPI1_PCS0),
+		{
+			0,
+			{
+				PIN2MUX(SPI1_SCK),
+				PIN2MUX(SPI1_SD0),
+				PIN2MUX(SPI1_SDI),
+#if defined(SPI1_PCS0) && !ISEMPTY(SPI1_PCS0)
+				PIN2MUX(SPI1_PCS0),
 #endif
-#ifdef SPI1_PCS1
-		PIN2MUX(SPI1_PCS1),
+#if defined(SPI1_PCS1) && !ISEMPTY(SPI1_PCS1)
```

Comment:
seems like  `defined` should be inside `isempty` as it's always used in pair here?

Reply:
This would be undefined behavior (see `-Wexpansion-to-defined`).